src/vstart.sh

```diff
@@ -1,5 +1,28 @@
 #!/bin/sh
 
+# abort on failure
+set -e
+
+if [ -e CMakeCache.txt ]; then
+  # Out of tree build, learn source location from CMakeCache.txt
+  SRC_ROOT=`grep Ceph_SOURCE_DIR CMakeCache.txt | cut -d "=" -f 2`
+  [ -z "$PYBIND" ] && PYBIND=$SRC_ROOT/src/pybind
+  [ -z "$CEPH_ADM" ] && CEPH_ADM=./ceph
+  [ -z "$INIT_CEPH" ] && INIT_CEPH=./init-ceph
+  [ -z "$CEPH_BIN" ] && CEPH_BIN=src
+  [ -z "$CEPH_LIB" ] && CEPH_LIB=src
+  [ -z "$OBJCLASS_PATH" ] && OBJCLASS_PATH=src/cls
+
+  # Gather symlinks to EC plugins in one dir, because with CMake they
+  # are built into multiple locations
+  mkdir -p ec_plugins
+  for file in ./src/erasure-code/jerasure/*.so* src/erasure-code/lrc/*.so*
```

Comment:
can we simply put:

```sh
for file in src/erasure-code/*/libec_*.so*;
```

instead of enumerating all the erasure plugin directories?

since, with the fix (https://github.com/oritwas/ceph/commit/ed2fdd93288e17f84e23a96ff697f7f0bd963c20) by @oritwas and probably future erasure plugins, we need always keep this file in sync.